hail/src/main/scala/is/hail/expr/ir/IR.scala

```diff
@@ -229,6 +229,11 @@ final case class ArrayRef(a: IR, i: IR, errorID: Int) extends IR
 final case class ArraySlice(a: IR, start: IR, stop: Option[IR], step:IR = I32(1), errorID: Int = ErrorIDs.NO_ERROR) extends IR
 final case class ArrayLen(a: IR) extends IR
 final case class ArrayZeros(length: IR) extends IR
+
+object StreamRange {
+  def iota(requiresMemoryManagement: Boolean = false): StreamRange = StreamRange(0, Int.MaxValue, 1, requiresMemoryManagement)
```

Comment:
Missed this and `zipWithIndex`, but it's fine to just fix in the follow-up PR.

Reply:
oops, thanks!